pkg/providers/instancetype/instancetypes.go

```diff
@@ -269,11 +269,11 @@ func (p *DefaultProvider) getInstanceTypes(ctx context.Context) (map[string]*ske
 	}
 
 	skus := cache.List(ctx, skewer.IncludesFilter(GetKarpenterWorkingSKUs()))
-	log.FromContext(ctx).V(1).Info(fmt.Sprintf("Discovered %d SKUs", len(skus)))
+	log.FromContext(ctx).WithValues("skuCount", len(skus)).V(1).Info("discovered SKUs")
```

Comment:
Hmm, Yes, I think I prefer "skuCount" over more generic "count"; again wondering if there are best practices for this ...